physionet-django/user/fixtures/demo-user.json

```diff
@@ -14332,6 +14365,11 @@
         "view_event_menu",
         "user",
         "event"
+      ],
```

Comment:
The "Host" is not an admin role (the group simply allows people able to create events) so I don't think this group should be able to view the admin console.

Reply:
Noted. Thanks. Will make the changes to fixtures